plugin-gradle/common/src/main/kotlin/org/jsonschema2dataclass/internal/task/Js2dGeneratorTaskBase.kt

```diff
@@ -23,11 +23,11 @@ abstract class Js2dGeneratorTaskBase<ConfigType> @Inject constructor(
     abstract var uuid: UUID
 
     @get:Classpath
-    val js2dConfiguration: NamedDomainObjectProvider<Configuration> =
+    val js2dConfiguration: Provider<out FileCollection> =
```

Comment:
When this configuration item is loaded from the cache Gradle tries to assign a provider of file collection which fails because the type used here is more specific than that. There is no need to make it that specific, though, because no code depends on it being that specific and making it less specific makes the plugin configuration cache compatible.